tracer/test/snapshots/Security.AspNetCore5AsmActionsConfiguration.__type=block_request_statusCode=200.verified.txt

```diff
@@ -17,8 +17,8 @@
       http.request.headers.host: localhost:00000,
       http.request.headers.user-agent: Mistake Not...,
       http.request.headers.x-forwarded-for: 86.242.244.246,
-      http.response.headers.content-length: 0,
-      http.status_code: 302,
+      http.response.headers.content-type: application/json,
+      http.status_code: 403,
```

Comment:
This commit, reverts changes from 98dfb3d back to the code in master: https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/snapshots/Security.AspNetCore5AsmActionsConfiguration.__type%3Dblock_request_statusCode%3D200.verified.txt#L21

Overall the only changes to this file are the new git tags, so this change can be safely ignored.

Reply:
Oh, weird, thanks! 😅